godot-core/src/builtin/quaternion.rs

```diff
@@ -30,10 +30,14 @@ impl Quaternion {
         Self { x, y, z, w }
     }
 
+    /// Creates a Quaternion from a Vector3 and an angle.
+    ///
+    /// # Panics
+    /// If the Vector3 is not normalized.
     pub fn from_angle_axis(axis: Vector3, angle: real) -> Self {
```

Comment:
Btw, while you're at it -- could you rename this function to `from_axis_angle`?

I don't know why we chose `from_angle_axis`, but given the parameter order, it makes not much sense. There is also [`Basis::from_axis_angle`](https://godot-rust.github.io/docs/gdext/master/godot/builtin/struct.Basis.html#method.from_axis_angle).